src/app/services/speech_recognition.py

```diff
@@ -0,0 +1,92 @@
+import requests
+from typing import Union, Literal, Optional
+from dataclasses import dataclass
+
+from jinja2 import Environment, PackageLoader
+
+from app.services.base import Service, ContainerConfig
+from app.job import LocalJobConfig, SlurmJobConfig, EC2JobConfig
+from app.logger import logger
+
+
+# NOTE: TextGeneration container is optimized for NVIDIA A100, A10G and T4 GPUs with
+# CUDA 12.2+ and requires NVIDIA Container Toolkit on the service host. The image
+# was built to run on GPU and will not reliably work without GPU support.
+
+
+SpeechRecognitionModels = {
+    "openai/whisper-large-v3": {},
+    "openai/whisper-tiny": {},
+}
+
+
+# The container options which are needed when setting up
+# service API. These options are not in job.py
+@dataclass
+class SpeechRecognitionConfig(ContainerConfig):
+    revision: Optional[str] = None
+    port: Optional[int] = None
+
```

Comment:
Add `input_dir: str`. I guess this isn't optional? Unless the API defaults to looking in the root dir?

Reply:
This is not optional so users need to specify the location of their input files.